WcaOnRails/config/locales/en.yml

```diff
@@ -383,6 +384,7 @@ en:
         competitor_limit_reason: "What is the reason for the limit on competitors? Please fill this out in English!"
         venueAddress: ""
         championship_type: ""
+        registration_requirements: "Please indicate here any specificities about the registration process, as needed per <a href=\"https://www.worldcubeassociation.org/regulations/guidelines.html#2c1+\" target=\"_blank\">Guideline 2c1+</a>. After the competition is announced, this field will <b>not</b> be editable anymore, and Board's approval will be needed to modify it.<br/>For example indicate here if competitors need to have a clean payment history with a specific entity, if there is some refund policy, or if they will have to answer a confirmation email in case the maximum number of registrations is reached.<br/>Please fill these information <b>at least in English</b>; it will be displayed on the competition's registration page if you are using the WCA website registrations."
```

Comment:
"Please fill these information *at least in English*" feels awkward to me. Maybe something like this instead?

"Please fill this out in English and whatever other languages your competitors speak"